contract-tests/images/applications/botocore/botocore_server.py

```diff
@@ -10,7 +10,7 @@
 import requests
 from botocore.client import BaseClient
 from botocore.config import Config
-from typing_extensions import override
+from typing_extensions import Tuple, override
```

Comment:
Isn't this Tuple deprecated?
https://docs.python.org/3.12/library/typing.html#typing.Tuple

Reply:
It's deprecated since python 3.9, but there's a build for python 3.8 too, which will make the linter fail with this error: `E1136: Value 'tuple' is unsubscriptable (unsubscriptable-object)`